pkg/processor/build/builder.go

```diff
@@ -1075,10 +1075,11 @@ func (b *Builder) buildProcessorImage() (string, error) {
 			BuildTimeoutSeconds: b.resolveBuildTimeoutSeconds(),
 
 			// kaniko pod attributes
-			NodeSelector:      b.options.FunctionConfig.Spec.NodeSelector,
-			NodeName:          b.options.FunctionConfig.Spec.NodeName,
-			Affinity:          b.options.FunctionConfig.Spec.Affinity,
-			PriorityClassName: b.options.FunctionConfig.Spec.PriorityClassName,
+			NodeSelector:            b.options.FunctionConfig.Spec.NodeSelector,
+			NodeName:                b.options.FunctionConfig.Spec.NodeName,
+			Affinity:                b.options.FunctionConfig.Spec.Affinity,
+			PriorityClassName:       b.options.FunctionConfig.Spec.PriorityClassName,
+			ReadinessTimeoutSeconds: b.platform.GetConfig().FunctionReadinessTimeout,
```

Comment:
```suggestion
			ReadinessTimeoutSeconds: b.options.FunctionConfig.Spec.ReadinessTimeoutSeconds
```

Reply:
The functionconfig's readiness timeout can be 0 if not set, and is not really enriched.
There are a few places (such as the controller) that take the platformconfig's readiness timeout in case the above is 0, but not in this stage of the kaniko container builder.